### PR TITLE
GDAL 3.2.0

### PR DESCRIPTION
**Pull request by mjsteinbaugh**
Update GDAL to 3.2.0 from 3.1.4.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Update GDAL to 3.2.0 from 3.1.4.

https://github.com/OSGeo/gdal/releases/tag/v3.2.0

Seeing this with `brew test gdal`:

```
❯ brew test gdal
==> Testing gdal
==> /usr/local/Cellar/gdal/3.2.0/bin/gdalinfo --formats
==> /usr/local/Cellar/gdal/3.2.0/bin/ogrinfo --formats
==> /usr/local/opt/python@3.9/bin/python3 -c import gdal
Last 15 lines from /Users/mike/Library/Logs/Homebrew/gdal/test.03.python3:
2020-11-02 09:49:33 -0500

/usr/local/opt/python@3.9/bin/python3
-c
import gdal

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'gdal'
Error: gdal: failed
An exception occurred within a child process:
  BuildError: Failed executing: /usr/local/opt/python@3.9/bin/python3 -c import\ gdal
/usr/local/Homebrew/Library/Homebrew/formula.rb:2022:in `block in system'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1960:in `open'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1960:in `system'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/gdal.rb:169:in `block in <class:Gdal>'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1831:in `block (3 levels) in run_test'
/usr/local/Homebrew/Library/Homebrew/utils.rb:497:in `with_env'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1830:in `block (2 levels) in run_test'
/usr/local/Homebrew/Library/Homebrew/formula.rb:901:in `with_logging'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1829:in `block in run_test'
/usr/local/Homebrew/Library/Homebrew/mktemp.rb:58:in `block in run'
/usr/local/Homebrew/Library/Homebrew/mktemp.rb:58:in `chdir'
/usr/local/Homebrew/Library/Homebrew/mktemp.rb:58:in `run'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2071:in `mktemp'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1823:in `run_test'
/usr/local/Homebrew/Library/Homebrew/test.rb:43:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:93:in `block in timeout'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `block in catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:108:in `timeout'
/usr/local/Homebrew/Library/Homebrew/test.rb:42:in `<main>'
```

### CODE REVIEW

**Comment by mjsteinbaugh** [comment]
Not sure how to fix this. Any ideas?

**Comment by SMillerDev** [comment]
Does the changelog mention removing python support?

**Comment by mjsteinbaugh** [comment]
Digging in, I think it could be this:

https://github.com/OSGeo/gdal/blob/v3.2.0/gdal/NEWS#L23

"Python bindings: move implementation of scripts (except gdal2tiles) in osgeo.utils package to be reusable"

**Comment by mjsteinbaugh** [comment]
Seems like it's these lines in question in `gdal.rb`:

```rb
    # Build Python bindings
    cd "swig/python" do
      system Formula["python@3.9"].opt_bin/"python3", *Language::Python.setup_install_args(prefix)
    end
    bin.install Dir["swig/python/scripts/*.py"]
```

```rb
    system Formula["python@3.9"].opt_bin/"python3", "-c", "import gdal"
```

**Comment by SMillerDev** [comment]
I think it just needs to be 
```ruby
    system Formula["python@3.9"].opt_bin/"python3", "-c", "import osgeo.utils.gdal"
```

**Comment by mjsteinbaugh** [comment]
@SMillerDev I confirm the `osgeo.gdal` change works on my machine.

```sh
❯ brew test gdal
==> Testing gdal
==> /usr/local/Cellar/gdal/3.2.0/bin/gdalinfo --formats
==> /usr/local/Cellar/gdal/3.2.0/bin/ogrinfo --formats
==> /usr/local/opt/python@3.9/bin/python3 -c import osgeo.gdal
```

**Comment by SMillerDev** [comment]
So it's fine with gdal now. There's just errors in dependencies. For all the formula that say "image not found" in the CI log you should run `brew bump-revision -m "gdal" Formula`

**Comment by mjsteinbaugh** [comment]
Checks are failing because of a URL timeout:

```
==> FAILED
Error: 1 problem in 1 formula detected
simple-tiles:
  * The URL https://propublica.github.io/simple-tiles/ is not reachable (HTTP status code 404)
```

**Comment by SMillerDev** [comment]
And `gmt` has a duplicate revision bump. Can you check if you can still reach the simple-tiles page personally?

File: Formula/gmt.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Gmt < Formula
   mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.1.1-src.tar.xz"
   sha256 "d476cba999340648146ef53ab4a3f64858cbd2f5511cdec9f7f06f3fb7896625"
   license "LGPL-3.0-or-later"
+  revision 2
```
Comment (SMillerDev) [inline Formula/gmt.rb:8]:
```suggestion
  revision 1
```

**Review by SMillerDev — COMMENTED** [review]
If you can't acces the simple-tiles page, we should switch the homepage url to their github repo.

**Comment by mjsteinbaugh** [comment]
Thanks, yeah `https://github.com/propublica/simple-tiles` is accessible. I'll work on changing that.

**Comment by mjsteinbaugh** [comment]
Looks like it's safe to merge now.

**Comment by SMillerDev** [comment]
Could you please rebase and make it one commit per formula? Mostly the merge commit in between messes with the merge bot.

**Comment by mjsteinbaugh** [comment]
I'm opening a fresh pull request with a simpler commit history.

**Pull request closed without merging**